sources/packages/web/src/views/aest/StudentApplication.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <header-navigator
```

Comment:
this file is actually renamed from `ApplicationDetails.vue` to `StudentApplication.vue`

Reply:
This is the page that will become a component in the upcoming tickets, right?

Reply:
yes,